doc/api/stream.md

```diff
@@ -219,13 +219,19 @@ added: v0.9.4
 ##### Event: 'close'
 <!-- YAML
 added: v0.9.4
+changes:
+  - version: v10.0.0
+    pr-url: https://github.com/nodejs/node/pull/18438
+    description: >
```

Comment:
stream.md seems to be the only file using this `>` YAML markup syntax, I suggest using
```
    description: Add `emitClose` option to specify if `'close'` is emitted
                        on destroy.
```

I'm guessing at the 80 column margin above, and with the github font, I can't tell if the "Add" and the "on" are aligned, but I hope its clear. That's how all the other API docs wrap long descriptions.

nit: a trailing `.` makes it a sentence.

Reply:
This is a copy-paste from another place (the constructors). 

```
      Add `emitClose` option to specify if `'close'` is emitted on destroy
```

I prefer not changing this sentence, otherwise we should change it everywhere.

Note that his passes linting, so it should be good to go.